orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -401,13 +401,11 @@ Status TrainingRunner::PrepareFeedNamesAndFeeds(const SessionMode mode,
     ORT_ENFORCE(params_.pipeline_parallel_size > 1);
     feed_names.push_back(pipeline_context_.forward_waited_event_name);
     OrtValue event_id;
-    const int64_t id = (mode == EvaluateStep) ? -1 : pipeline_schedule_.GetForwardWaitedEventId(
-      pipeline_context_.pipeline_stage_id,
-      static_cast<int>(step_) % pipeline_context_.num_pipeline_batches);
+    const int64_t id = (mode == EvaluateStep) ? -1 : pipeline_schedule_.GetForwardWaitedEventId(pipeline_context_.pipeline_stage_id, static_cast<int>(step_) % pipeline_context_.num_pipeline_batches);
```

Comment:
line is too long, I thought it's recommended 120 char per line

Reply:
+1